viite-backend/viite-main/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala

```diff
@@ -3653,6 +3654,137 @@ class ProjectServiceSpec extends FunSuite with Matchers with BeforeAndAfter {
     project
   }
 
+   test("Test recalculateProjectLinks() When terminating last project links from two track section, creating a minor discontinuity on both tracks, then the tracks should be adjusted to match at the discontinuous part.") {
```

Comment:
When terminating last project links from two track section, *that are of different lengths,* ?